Fluffy EOL:  enterprise docs, replace fluffy for enterprise (#9204)
treeverse/lakeFS: docs/src/enterprise/getstarted/migrate-from-oss.md

Reviewer: Let's verify those steps are still correct, literally hands on testing.
Make sure the upgrade path is very clear, not missing steps and not having extra steps here that are already written. 
If you'd like I'd be happy to help you test this and share more context f2f.

To migrate from lakeFS Open Source to lakeFS Enterprise, follow the steps below:

> 1. Make sure you have the lakeFS Enterprise Docker token, if not [contact us](https://lakefs.io/contact-sales/) to gain access to lakeFS Enterprise. You will be granted with a token that enables downloading *dockerhub/lakeFS-Enterprise* from [Docker Hub](https://hub.docker.com/u/treeverse).

Reviewer 2: ```suggestion
1. Make sure you have the lakeFS Enterprise Docker token. if not, [contact us](https://lakefs.io/contact-sales/) to gain access to lakeFS Enterprise. You will be granted a token that enables downloading *dockerhub/lakeFS-Enterprise* from [Docker Hub](https://hub.docker.com/u/treeverse).
```

> 1. Update lakeFS docker image to enterprise. Use `treeverse/lakefs-enterprise` instead of `treeverse/lakefs`. The image can be pulled using the lakeFS Enterprise token.

Reviewer 2: ```suggestion
1. Update the lakeFS Docker image to the enterprise version. Replace `treeverse/lakefs` with `treeverse/lakefs-enterprise` in your configuration. The enterprise image can be pulled using your lakeFS Enterprise token.
```

1. Sanity Test (Optional): Install a new test lakeFS Enterprise before moving your current production setup. Test the setup > login > Create repository etc. Once everything seems to work, delete and cleanup the test setup and we will move to the migration process.
1. Follow lakeFS [Enterprise installation guide][lakefs-enterprise-install]
    1. Make sure that you meet the [prerequisites][lakefs-enterprise-install-prerequisites]
    1. Update your existing `values.yaml` file for your deployment

> 1. DB Migration: we are going to use the same DB for lakeFS Enterprise, so we need to migrate the DB schema.

Reviewer 2: ```suggestion
1. DB Migration: We are going to use the same database for lakeFS Enterprise, so we need to migrate the database schema. 
```

To migrate from lakeFS Open Source to lakeFS Enterprise, follow the steps below:

> 1. Make sure you have the lakeFS Enterprise Docker token. if not, [contact us](https://lakefs.io/contact-sales/) to gain access to lakeFS Enterprise. You will be granted a token that enables downloading *dockerhub/lakeFS-Enterprise* from [Docker Hub](https://hub.docker.com/u/treeverse).

Reviewer: Dockerhub repos are lower case only, also be consistent with docs 
```suggestion
1. Make sure you have the lakeFS Enterprise Docker token. if not, [contact us](https://lakefs.io/contact-sales/) to gain access to lakeFS Enterprise. You will be granted a token that enables downloading *dockerhub/lakeFS-enterprise* from [Docker Hub](https://hub.docker.com/u/treeverse).
```

1. Update the lakeFS Docker image to the enterprise version. Replace `treeverse/lakefs` with `treeverse/lakefs-enterprise` in your configuration. The enterprise image can be pulled using your lakeFS Enterprise token.
1. Sanity Test (Optional): Install a new test lakeFS Enterprise before moving your current production setup. Test the setup > login > Create repository etc. Once everything seems to work, delete and cleanup the test setup and we will move to the migration process.

> 1. Follow lakeFS [Enterprise installation guide][lakefs-enterprise-install]

Reviewer: This step seems random in the context but the reference to install guide for enterprise is important. 
perhaps rephrase to say "Make sure to configure the enterprise properly, see [Enterprise installation guide][lakefs-enterprise-install]" and that;s it.

>     1. Make sure that you meet the [prerequisites][lakefs-enterprise-install-prerequisites]

Reviewer: Remove. pre-requisites are already inside and this is irrelevant, only for people with k8s.

>     1. Configure `features.local_rbac = true`

Reviewer: Remove this line, it's the default and could fail if running with oss binary.

>     1. Update your existing `values.yaml` file for your deployment

Reviewer: ```suggestion
    1. Update your existing lakeFS configuration for enterprise (i.e `values.yaml` file for your deployment if using helm)
```
